config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+Xh84L3iV3L+XdDLLIhddxjrMDGzvnp7FIGegxr2WzGOU37OkY5uK5NmQiQhGI5hkFJQLg5x7RUsKH6xFaxSs8jJBzTbpkEB7ZbCUgLBokyUmLiRvTljOAZWScUzLDd9ZL+mVNnojL0zevoxY+VlJhDnbWltGfMEXJ/fcsE4Wjlsdh3mq0KT6ihfz18bsVthXLoCr0DtD1MWxsSQVRpiGbc4YjeP8FHZqxKK/nNHZ1ro/b0OOZcwRjyGQRI4Z238TNPp64YED6TeCKiGwoOplIM677ADdFgo7tTO5Tf9SnKfpzZ7fKSvE+2K/v0uK7CAYEwmk4GetDMuorufWBFvSf8HSLWfhpeCoe9DJFY5F/M711jTXgEeQta18IR0n7QHtyPBV56VR/FAt8+1IVGr1pHcnHj4/EyORGbrk--TJ5bKNd6jR/UIf9m--U3lT0rEk2CtT8rYljAk7aQ==
```

Comment:
create `credentials.yml.enc.example` and push to git, this file add to gitignore